force-app/main/default/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -68,6 +68,7 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
     @track hasInvalidBatchFields = false;
     @track missingBatchHeaderFieldLabels = [];
     @track missingRequiredFieldsMessage;
+    @track allowFirstInstallment = false;
```

Comment:
These changes fix the issues in Edit Batch Info where "First installment paid" is not disabled and/or unchecked when "Allow Recurring Donations" is changed.